docs/modules/clientutils.rst

```diff
@@ -214,6 +214,25 @@ Retrieves boundaries for all DOM element matching the provided :ref:`selector <s
 
 It returns an array of objects each having four keys: ``top``, ``left``, ``width`` and ``height``.
 
+.. index:: Link text
+
+``getElementByLinkText()``
+-------------------------------------------------------------------------------
+
+**Signature:** ``getElementByLinkText(String text [, HTMLElement scope])``
+
+Retrieves a single DOM element matching a given :ref:`Link text <selectors>`.
+
+.. versionadded:: 1.0
```

Comment:
Perhaps all there "versionadded" parts should have unified version number in them and the way it would seem to be now, it is `1.1`.